### ISSUE TITLE
Need contact points exposed on collision Response object

### ISSUE THREAD

**Issue by Nashorn** [jriecken/sat-js#63]
Need contact points exposed on collision Response object. Needed for physics

**Comment by jriecken** [comment]
I agree - unfortunately I currently don't have time to spend on any significant changes to this library. I would take a look at a pull request though (maybe exposed as a method on the `SAT.Response` object?

It should be possible to compute a contact manifold (e.g in our case either a point or a line) using a method like this: http://www.dyn4j.org/2011/11/contact-points-using-clipping/ - the `overlapN` and `overlap` in the `SAT.Response` are the "separation normal" and "depth" that the article talks about getting back from the collision detection phase.
